frontend/src/app/shared/header/header.component.html

```diff
@@ -14,5 +14,6 @@
       </a>
     </div>
   </div>
+  <a (click)="giveFeedback()" id="give-feedback" class="cursor-pointer">Give feedback</a>
```

Comment:
why switch from `submit` to `give`? stick to the same term throughout it's usage

Reply:
`giveFeedback()` was replaced by usage of `routerLink`